documentation/book/con-what-the-cluster-operator-does.adoc

```diff
@@ -1,34 +1,31 @@
 // Module included in the following assemblies:
 //
 // assembly-operators-cluster-operator.adoc
-// assembnly-cluster-operator.adoc
+// assembly-cluster-operator.adoc
 
 [id='con-what-the-cluster-operator-does-{context}']
 = Overview of the Cluster Operator component
 
-The Cluster Operator is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble.
-As part of the Kafka cluster, it can also deploy the topic operator which provides operator-style topic management via `KafkaTopic` custom resources.
-The Cluster Operator is also able to deploy a Kafka Connect cluster which connects to an existing Kafka cluster.
-On OpenShift such a cluster can be deployed using the Source2Image feature, providing an easy way of including more connectors.
+The Cluster Operator can deploy clusters for:
 
-.Example architecture for the Cluster Operator
+* Kafka (including Zookeeper)
+* Kafka Connect
+* Kafka Mirror Maker
+* Kafka Bridge
 
-image:cluster-operator.png[Cluster Operator]
+The Cluster Operator can also deploy (through Entity Operator configuration)
```

Comment:
Should we make it clear that this is only with the `Kafka` resource?

Reply:
Definitely. I've reworded and rearranged to make more clear.